nibiru/assetlist.json

```diff
@@ -25,7 +25,7 @@
       ],
       "base": "unibi",
       "name": "Nibiru",
-      "display": "nibi",
+      "display": "NIBI",
```

Comment:
This unit is not defined and will not pass tests. It was fine before, and I assume you mistake what the 'display' property means. "Symbol" is shown to users, while "display" is only used to determine the number decimal points for the asset amounts.

Reply:
I see, so the display has to match one of the denom units, while symbol is the actual string that people see in most wallets

Fixed in 5f8a4e8ee8cfceca0a8d914b5623e3876f0175ff